peernode/src/peer-node/producer/mod.rs

```diff
@@ -44,28 +62,25 @@ pub async fn run(market: String, ip: Option<String>, port: Option<u16>) -> Resul
     };
     println!("Producer: IP address is {}", ip);
 
-    // Generate a random producer ID
+    // Generate a random Producer ID
     let producer_id = uuid::Uuid::new_v4().to_string();
 
-    // Register the files with the market service
-    let hash = file_map.get_hashes().await;
-    for hash in hash {
-        // TODO: Find a way to get the public IP address
-        println!("Producer: Registering file with hash {}", hash);
+    for (hash, price) in prices {
+        println!(
+            "Producer: Registering file with hash {} and price {}",
+            hash, price
+        );
```

Comment:
The println is short enough to fit in one line right?

Reply:
<img width="788" alt="Screenshot 2024-04-15 at 6 43 16 PM" src="https://github.com/sbu-416-24sp/orcanet-rust/assets/54873722/c785e685-8101-40e1-bc7c-2c296ae80dec">

Reply:
I meant to fit in one line in the code.
`println!("Producer: Registering file with hash {} and price {}", hash, price);`

Reply:
This is using the auto formatter built into cargo "cargo fmt"